src/probnum/filtsmooth/gaussfiltsmooth/kalman.py

```diff
@@ -57,23 +107,27 @@ def iterated_filtsmooth(
         if stopcrit is None:
             stopcrit = StoppingCriterion()
 
-        # Initialise iterated smoother
-        old_posterior = self.filtsmooth(
-            regression_problem,
-            _previous_posterior=None,
-        )
-        new_posterior = old_posterior
+        info_dicts = []
+        if init_posterior is None:
+            # Initialise iterated smoother
+            init_posterior, info_dicts = self.filtsmooth(
+                regression_problem,
+                _previous_posterior=None,
+            )
+
+        new_posterior = init_posterior
```

Comment:
just checking: so you decided against the "else" solution here? :)

Reply:
Nope, forgot, thanks for the reminder! Just one sec